### PR TITLE
[KOGITO-271] Upgrade to SDK 0.10.0

### PR DESCRIPTION
**Pull request by ricardozanini**
See: https://issues.jboss.org/browse/KOGITO-271

In this PR:

1. Simple upgrade to Operator SDK 0.10.0
2. Since this is a major change that can mess things up, we fixed the e2e tests to make sure everything is working cluster wide. It's darn simple, but should be improved in the future (See https://issues.jboss.org/browse/KOGITO-130)

Many thanks for submiting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

- [x] You have read the [contributors guide](CONTRIBUTING.MD#sending-a-pull-request)
- [x] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [x] Your feature/bug fix has a unit test that verifies it
- [x] You've tested the new feature/bug fix in an actual OpenShift cluster

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -459,6 +459,18 @@ It's possible to verify the operator status by running:
 $ oc describe operatorsource.operators.coreos.com/kogitocloud-operator -n openshift-marketplace
 ```
 
+#### Running End to End tests
+
+If you have an OpenShift cluster and admin privileges, you can run e2e tests with the following command:
+
+```bash
+$  ./hack/run-e2e.sh <namespace> <tag>
```
Comment (xiezhang7) [inline README.md]: I think we can create a task in the makefile for the e2e testing.

**Comment by ricardozanini** [comment]
> Thanks, @ricardozanini . Since v0.10.0 supports [configuring timestamp format for logging](https://github.com/operator-framework/operator-sdk/pull/1529), I think we should make the timestamp more readable.

Hmm... the logs showed with the  `DEBUG` flag turned on, you mean? Otherwise those logs are meant to be consumed by the OpenShift log stack, which means that his format should be valid there. Correct? If not, we can add this change for sure.

File: README.md (hunk 1 above)
Comment (ricardozanini) [reply README.md]: I was thinking about that. I guess won't hurt adding to the Makefile

**Comment by ricardozanini** [comment]
@xiezhang7 I added the iso format to our inner log, but this change on 0.10.0 it's not meant to be used outside their CLI, as far as I understood that PR you mentioned.

Anyway, now the log is much more readable:

```
{"level":"info","T":"2019-09-17T20:07:14.689Z","logger":"controller_kogitoapp","msg":"Checking if build for 'example-quarkus' is finished"}
{"level":"info","T":"2019-09-17T20:07:14.801Z","logger":"controller_kogitoapp","msg":"Reconcile for 'example-quarkus' successfully finished"}
{"level":"info","T":"2019-09-17T20:07:14.801Z","logger":"controller_kogitoapp","msg":"Reconciling KogitoApp"}
{"level":"info","T":"2019-09-17T20:07:14.801Z","logger":"controller_kogitoapp","msg":"Checking if all resources for 'karina-test' are created"}
{"level":"info","T":"2019-09-17T20:07:14.815Z","logger":"controller_kogitoapp","msg":"Checking if build for 'karina-test' is finished"}
{"level":"info","T":"2019-09-17T20:07:14.927Z","logger":"controller_kogitoapp","msg":"Reconcile for 'karina-test' successfully finished"}
{"level":"info","T":"2019-09-17T20:07:14.928Z","logger":"controller_kogitoapp","msg":"Reconciling KogitoApp"}
{"level":"info","T":"2019-09-17T20:07:14.928Z","logger":"controller_kogitoapp","msg":"Checking if all resources for 'jbpm-quarkus-example' are created"}
{"level":"info","T":"2019-09-17T20:07:14.939Z","logger":"controller_kogitoapp","msg":"Checking if build for 'jbpm-quarkus-example' is finished"}
{"level":"info","T":"2019-09-17T20:07:15.053Z","logger":"controller_kogitoapp","msg":"Reconcile for 'jbpm-quarkus-example' successfully finished"}
```

Could you please review?

**Comment by xiezhang7** [comment]
> @xiezhang7 I added the iso format to our inner log, but this change on 0.10.0 it's not meant to be used outside their CLI, as far as I understood that PR you mentioned.
> 
> Anyway, now the log is much more readable
> 
> Could you please review?

Thanks, @ricardozanini . It looks great.

**Pull request merged**